Lib/pathlib/_local.py

```diff
@@ -780,6 +784,19 @@ def mkdir(self, mode=0o777, parents=False, exist_ok=False):
             if not exist_ok or not self.is_dir():
                 raise
 
+    if hasattr(_winapi, 'CopyFile2'):
+        def copy(self, target, follow_symlinks=True):
+            """
+            Copy the contents of this file to the given target. If this file is a
+            symlink and follow_symlinks is false, a symlink will be created at the
+            target.
+            """
+            if isinstance(target, PathBase) and not isinstance(target, Path):
```

Comment:
Is there a way to replace this with checking for `os.fspath` failures? In theory,[^1] if the objects support `fspath`, then that means we can use them as a source and target for a file system path?

[^1]: I could be wrong, going entirely off memory here. But I'm pretty sure `fspath` is meant for _file system_ paths, not just any path.

Reply:
How's it look now?